shadowsocks-csharp/View/MenuViewController.cs

```diff
@@ -104,6 +108,11 @@ public MenuViewController(ShadowsocksController controller)
             }
         }
 
+        private void _notifyIcon_MouseMove(object sender, MouseEventArgs e)
+        {
+            UpdateTrayIcon();

```

Comment:
You would like to redraw the icon just in case user change the theme color or display DPI.
This causes redundant paintings when user move mouse on the icon, pixel by pixel.

Reply:
Moved UpdateTrayIcon to notifyIcon1_Click.